lib/Wikia/src/Service/User/Attributes/UserAttributes.php

```diff
@@ -78,30 +70,18 @@ private function loadAttributes( $userId ) {
 			return $this->attributes[$userId];
 		}
 
-		$attributes = $this->loadFromMemcache( $userId );
-		if ( $attributes === false ) {
-			$attributes = [];
-			/** @var Attribute $attribute */
-			foreach ( $this->attributeService->get( $userId ) as $attribute ) {
-				$attributes[$attribute->getName()] = $attribute->getValue();
-			};
-			$this->logAttributeServiceRequest( $userId );
-			$this->setInMemcache( $userId, $attributes );
-		}
+		$attributes = [];
+		/** @var Attribute $attribute */
+		foreach ( $this->attributeService->get( $userId ) as $attribute ) {
+			$attributes[$attribute->getName()] = $attribute->getValue();
+		};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1116)